linux_sql/README.md

```diff
@@ -1 +1,47 @@
-testing github
+## Introduction
+The Cluster Monitor Agent is an internal tool that collects the hardware information of each server and monitoros resource usage such as CPU and RAM in real time. The data will be stored in a RDBMS database on a desinated server.  The report generated by using these date will be a great tools for future recourse planning.
+
+## Architecture and Design
+1 Cluster Monitor Agent Diagram:
+![image](/linux_sql/assets/sizedDiagram.png)
```

Comment:
all arrows should point to the psql
![image](https://user-images.githubusercontent.com/58951265/77783694-b2936780-702f-11ea-989d-61812a840d1b.png)